src/PermissionManager.sol

```diff
@@ -240,8 +240,6 @@ contract PermissionManager is IERC1271, Ownable2Step, Pausable {
 
     /// @notice Revoke a permission to disable its use indefinitely.
     ///
-    /// @dev Depending on permission contract implementation, permissions can revoke other permissions.
-    ///
     /// @param permissionHash hash of the permission to revoke
```

Comment:
Finding #4 recommended to add a note here that says something along the lines of "Once a permission is revoked, it cannot be re-enabled. A new permission will need to be approved."

should we address it here for clarity?

Reply:
I saw the comment was added in the docs, rather than at the revokePermission() function, I'll resolve this comment